scripts/image-builder/config/kickstart.ks.template

```diff
@@ -14,13 +14,15 @@ network --bootproto=dhcp --device=link --activate --onboot=on
 #
```

Comment:
Need to fix a comment here as well. Let's keep it as short as possible though.

Reply:
done.